t/dotfiles.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/local/bin/perl
```

Comment:
When adding a new file, you also need to update `MANIFEST`. That's the reason that the Travis CI builds are currently failing for this pull request.